source/_integrations/lifx.markdown

```diff
@@ -124,9 +128,35 @@ Run an effect with colors looping around the color wheel. All participating ligh
 | `spread` | Maximum color difference between participating lights, in degrees on a color wheel (ranges from 0 to 359).
 | `power_on` | Set this to False to skip the effect on lights that are turned off (defaults to True).
 
+### Service `lifx.effect_flame`
+
+Run a hardware-based effect on LIFX matrix devices that creates a flame effects on the device. The speed of the flames is controlled by the `speed` attribute. The device will be powered on by default, but this can be overridden by setting `power_on` to `false`.
```

Comment:
```suggestion
Run a hardware-based effect on LIFX matrix devices that creates a flame effect on the device. The device will be powered on by default, but this can be overridden by setting `power_on` to `false`. The `speed` attribute controls the speed of the flames.
```